src/ui/molecules/ToggleComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode } from 'react';
+import { ContentToggle, SelectField } from '../atoms';
+
+export interface Value {
```

Comment:
This code is duplicated from the SelectField, so if in the future the interface needed updating the change would need to be made in 2 places. How could this be changed to ensure that there is only a single point of truth?